### PR TITLE
add formatDate for convert date

### PR DESCRIPTION
**Pull request by KarimQaderi**
## Issue Reference
https://github.com/bagisto/bagisto/issues/6867

## Description
I need to be able to convert date to Persian date

### CODE REVIEW

File: packages/Webkul/Core/src/Core.php [hunk 1]
```diff
@@ -790,6 +790,10 @@ public function formatDate($date = null, $format = 'd-m-Y H:i:s')
             $date = Carbon::now();
         }
 
+        if (is_string($date)){
```
Comment (devansh-webkul) [inline packages/Webkul/Core/src/Core.php:793]: Add space after the bracket.

File: packages/Webkul/Ui/src/DataGrid/Traits/ProvideCollection.php [hunk 2]
```diff
@@ -253,6 +253,10 @@ private function transformColumns($record)
                 ) {
                     $record->{$column['index']} = $column['closure']($record);
                 }
+            } elseif ($column['type'] == 'datetime'){
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/DataGrid/Traits/ProvideCollection.php:256]: Same here.

File: packages/Webkul/Ui/src/DataGrid/Traits/ProvideCollection.php [hunk 3]
```diff
@@ -253,6 +253,10 @@ private function transformColumns($record)
...
+                $record->{$column['index']} = core()->formatDate($record->{$column['index']}, $column['format'] ?? 'Y-m-d H:i:s');
+            } elseif ($column['type'] == 'date'){
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/DataGrid/Traits/ProvideCollection.php:258]: Same here.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by KarimQaderi** [comment]
changes made as requested @devansh-webkul

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**